src/getDailySummary.js

```diff
@@ -4,7 +4,7 @@ import setup from './setup';
 import { get } from './services/apiKeyStore';
 import generateDailySummary from './services/generateDailySummary';
 
-const getDailySummary = async (date = new Date()) => {
+const getDailySummary = async (date = ''+new Date()) => {
```

Comment:
Does this break [this line](https://github.com/jaebradley/wakatime-cli/blob/master/src/getDailySummary.js#L16)?

I think we should keep `date` a `Date` and do any formatting in the `formattedDate` variable / create a helper method.